_pages/en_US/finalizing-setup.txt

```diff
@@ -169,7 +169,7 @@ You can now hold (Start) on boot to launch the Luma3DS chainloader menu (note th
 
 You can now hold (Start) + (Select) + (X) on boot to dump the ARM11 bootrom (`boot11.bin`), the ARM9 bootrom (`boot9.bin`), and your console unique OTP (`OTP.bin`) to the `/boot9strap/` folder on your SD card (note that this will not have any kind of prompt or message).
 
-You can now press (L) + (Down) + (Select) while the system is booted to open the Rosalina menu integrated into Luma3DS. For a full list of Rosalina features, please see the [Luma3DS v8.0 Release](https://github.com/AuroraWright/Luma3DS/releases/tag/v8.0)
+You can now press (L) + (Down) + (Select) while the system is booted to open the Rosalina menu integrated into Luma3DS. For a full list of Rosalina features, please see the [Luma3DS v8.0 Release](https://github.com/LumaTeam/Luma3DS/releases/tag/v8.0)
```

Comment:
switch link to https://github.com/LumaTeam/Luma3DS/wiki/Rosalina (probably out of scope for this PR)